src/test/java/org/jetbrains/bsp/bazel/BazelBspServerTest.java

```diff
@@ -83,7 +83,19 @@ public void run() {
             () ->
                 client.testScalaTestClasses(
                     BazelBspServerTestData.SCALA_TEST_CLASSES_PARAMS,
-                    BazelBspServerTestData.EXPECTED_SCALA_TEST_CLASSES))
+                    BazelBspServerTestData.EXPECTED_SCALA_TEST_CLASSES)),
+        new BazelBspServerSingleTest(
+            "javacopts test",
+            () ->
+                client.testJavacOptions(
+                    BazelBspServerTestData.JAVAC_OPTIONS_PARAMS,
+                    BazelBspServerTestData.EXPECTED_JAVAC_OPTIONS)),
+        new BazelBspServerSingleTest(
+            "scalacopts test",
+            () ->
+                client.testScalacOptions(
+                    BazelBspServerTestData.SCALAC_OPTIONS_PARAMS,
+                    BazelBspServerTestData.EXPECTED_SCALAC_OPTIONS))
```

Comment:
They should be commented out as well, since they are not passing yet. Wdyt?

Reply:
I just remembered that it makes more sense for us to just specify the version of bazel of the sample repository as one prior to 4.0.0 right off the bat :sweat_smile:

Reply:
I decided to set the version on `3.7.2` (most recent before `4.0.0`)